packages/web-components/src/components/masthead/left-nav-menu-section.ts

```diff
@@ -0,0 +1,265 @@
+    }
+  };
+
+  firstUpdated() {
+    if (this.sectionId === '-1, -1') {
+      this.expanded = true;
+      this.ariaHidden = 'false';
+    }
+  }
+
+  updated(changedProperties) {
+    if (changedProperties.has('expanded')) {
+      const { selectorNavMenu, selectorNavItem } = this.constructor as typeof DDSLeftNavMenuSection;
+      const { selectorTabbable: selectorTabbableForLeftNavMenuSection } = this.constructor as typeof DDSLeftNavMenuSection;
+      const { expanded, isSubmenu } = this;
+
+      if (expanded) {
+        if (isSubmenu) {
+          const backBtn = this.shadowRoot?.querySelector('button');
+          if (backBtn) {
+            backBtn.tabIndex = 0;
+          }
+        }
+        forEach(this.querySelectorAll(selectorNavMenu), elem => {
+          const item = (elem as HTMLElement).shadowRoot?.querySelector('button');
+          if (item) {
+            item.tabIndex = 0;
+          }
+        });
+        forEach(this.querySelectorAll(selectorNavItem), elem => {
+          const item = (elem as HTMLElement).shadowRoot?.querySelector('a');
+          if (item) {
+            item.tabIndex = 0;
+          }
+        });
+
+        // set focus to first element of menu panel to allow for tabbing through the menu
+        let tabbable;
+        if (isSubmenu) {
+          // set focus to back button
+          tabbable = this.shadowRoot?.querySelector('button');
+        } else {
+          // set focus to first menu item of section
+          tabbable = find(this.querySelectorAll(selectorTabbableForLeftNavMenuSection), elem =>
+            Boolean((elem as HTMLElement).offsetParent)
+          );
+        }
+
+        if (tabbable) {
+          document.addEventListener('transitionend', () => {
+            (tabbable as HTMLElement).focus();
+          });
+        }
+      } else {
+        forEach(this.querySelectorAll(selectorNavMenu), elem => {
+          const item = (elem as HTMLElement).shadowRoot?.querySelector('button');
+          if (item) {
+            item.tabIndex = -1;
+          }
+        });
+        forEach(this.querySelectorAll(selectorNavItem), elem => {
+          const item = (elem as HTMLElement).shadowRoot?.querySelector('a');
+          if (item) {
+            item.tabIndex = -1;
+          }
+        });
+        if (isSubmenu) {
+          const backBtn = this.shadowRoot?.querySelector('button');
+          if (backBtn) {
+            backBtn.tabIndex = -1;
+          }
+        }
+      }
+    }
+  }
+
+  render() {
+    const { backButtonText, title, _handleClickBack: handleClickBack, showBackBtn } = this;
+    return html`
+      <ul>
+        ${showBackBtn
+          ? html`
+              <li class="bx--side-nav__menu-item bx--masthead__side-nav--submemu-back" role="none">
+                <button class="bx--side-nav__link" tabindex="-1" role="menuitem" @click="${handleClickBack}">
+                  <span class="bx--side-nav__link-text">${ChevronLeft20()}${backButtonText}</span>
```

Comment:
Just to stay consistent with previous interpolation.

```suggestion
              <li class="${prefix}--side-nav__menu-item ${prefix}--masthead__side-nav--submemu-back" role="none">
                <button class="${prefix}--side-nav__link" tabindex="-1" role="menuitem" @click="${handleClickBack}">
                  <span class="${prefix}--side-nav__link-text">${ChevronLeft20()}${backButtonText}</span>
```